x-pack/platform/plugins/shared/fleet/server/routes/package_policy/handlers.ts

```diff
@@ -291,7 +291,7 @@ export const createPackagePolicyHandler: FleetRequestHandler<
   } catch (error) {
     appContextService
       .getLogger()
-      .error(`Error while creating package policy due to error: ${error.message}`);
+      .error(`Error while creating package policy due to error: ${error.message}\n${error.stack}`);
```

Comment:
nit: instead of adding the stack here we could add the error as part of LogMeta, and it will be available in JSON logs `.error(msg, { error})`